Shared.IntegrationTests/AdminApi/ManageFoldersTest.cs

```diff
@@ -67,5 +69,87 @@ public void TestFolderApi()
             Assert.AreEqual(1, deletionRes.Deleted.Count);
             Assert.AreEqual(subFolder1, deletionRes.Deleted[0]);
         }
+
+        [Test]
+        public void TestCreateFolder()
+        {
+            var testFolderName = $"{m_folderPrefix}_create_folder";
+
+            var createFolderResult = m_cloudinary.CreateFolder(testFolderName);
+
+            AssertCreateFolderResult(createFolderResult);
+
+            Thread.Sleep(2000);
+
+            var deletionResult = m_cloudinary.DeleteFolder(testFolderName);
+
+            AssertFolderDeletionAfterCreate(deletionResult, testFolderName);
+        }
+
+        [Test]
+        public async Task TestCreateFolderAsync()
+        {
+            var testFolderName = $"{m_folderPrefix}_create_folder_async";
+
+            var createFolderResult = await m_cloudinary.CreateFolderAsync(testFolderName);
+
+            AssertCreateFolderResult(createFolderResult);
+
+            Thread.Sleep(2000);
+
+            var deletionResult = await m_cloudinary.DeleteFolderAsync(testFolderName);
```

Comment:
What if folder is created, but `AssertCreateFolderResult` failed? 
This code is never executed and the folder is not deleted, junk is left on the test account.
Please move to `TearDown()`, since it is always executed. 
Please see other tests that perform cleanup.